Governance.md

```diff
@@ -0,0 +1,323 @@
+is more controversial than expected or a critical input is delayed because
+someone was unavailable, we can back out or further modify those changes based
+on the additional feedback.
+
+If necessary, a formal veto should consist of:
+
+-   an unambiguous statement that a veto is being invoked
+-   an explanation of why it is being invoked
+-   a description of what conditions (if any) would convince the vetoer to
+    withdraw their veto
+
+We expect that most people will take this responsibility seriously and only
+invoke their veto when they judge that a serious problem is being ignored and
+that their veto is necessary to protect The Project. We expect such vetoes to be
+rare in practice because Contributors are motivated from the start to find some
+solution that everyone can live with.
+
+If all proposals for resolving some issue are vetoed, then the status quo wins
+by default. In the worst case, if a Contributor is genuinely misusing their veto
+in an obstructive fashion to the detriment of the project, then they can be
+ejected from the project by consensus of the Steering Council – see below.
+
+For more information about the overall approach we use to achieve consensus, we
+highly recommend that all Contributors additionally read
+[Chapter 4: Social and Political Infrastructure](http://producingoss.com/en/producingoss.html#social-infrastructure)
+of Karl Fogel's classic Producing Open Source Software, and in particular the
+section on
+[Consensus-based Democracy](http://producingoss.com/en/producingoss.html#consensus-democracy),
+for a more detailed discussion.
+
+### Project Maintainers
+
+Project Maintainers are Project Contributors that have shown that they are
+dedicated to the ongoing development of the project through active engagement
+with the Community and facilitation of everyday Project activities. Maintainers
+are granted write access to one or more repositories within The Project, provide
+feedback on Github Issues and Discussions, review and merge Pull Requests,
+participate in monthly Community Meetings, vote on Steering Council members, and
+may be actively involved in setting the overall direction of the project.
+Project Maintainers are nominated by other Project Maintainers by sending a
+notification to the Project Maintainer's private mailing list. Other Project
+Maintainers will have a period of at least two weeks in which to veto a
+nomination; otherwise, the nominee shall become a Project Maintainer. No voting
+is necessary for selecting Project Maintainers.
+
+Project Maintainers who have not actively contributed to The Project for a
+period of 12 months will be asked if they wish to resign their write access and
+voting rights until they become active again.
+
+Project Maintainers may have their write access and voting rights revoked by the
+Steering Council if they are deemed to be actively harmful to The Project's
+well-being, and attempts at communication and conflict resolution have failed.
```

Comment:
Should that go through code of conduct committee? (or mention that this can also be a way for their rights to be revoked?)

Reply:
I think that is implicitly handled under `attempts at communication and conflict resolution` but we can make that more explicit by adding

```markdown
Depending on the nature of the conflict, if any, the Code of Conduct
Subcommittee may be involved in conflict resolution or specifically recommend
revoking a Maintainer's write access and voting rights. 
```

Reply:
I think that it is fair to explicitly mention that this should go through the CoC committee.